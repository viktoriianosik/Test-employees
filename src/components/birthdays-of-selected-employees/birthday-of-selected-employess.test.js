import React from "react";
import renderer from "react-test-renderer";
import BirthdaysOfSelectedEmployees from "./birthdays-of-selected-employees";
import testUsers from "../../test-data/test-data";


it(`Render component BirthdaysOfSelectedEmployees correctly`, () => {
  const tree = renderer.create(
      <BirthdaysOfSelectedEmployees
        selectedUsers={testUsers}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});