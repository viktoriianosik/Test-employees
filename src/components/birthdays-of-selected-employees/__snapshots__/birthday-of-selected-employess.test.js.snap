// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component BirthdaysOfSelectedEmployees correctly 1`] = `
<div
  className="employees-birthday"
>
  <h2
    className="employees-birthday__title title"
  >
    Employees birthday
  </h2>
  <h3>
    February
  </h3>
  <ul
    className="birthday-list"
  >
    <li
      className="birthday-item"
    >
      Bort Alex - 23 February, 2010 year
    </li>
  </ul>
  <h3>
    March
  </h3>
  <ul
    className="birthday-list"
  >
    <li
      className="birthday-item"
    >
      Nofl Mark - 29 March, 2014 year
    </li>
  </ul>
  <h3>
    June
  </h3>
  <ul
    className="birthday-list"
  >
    <li
      className="birthday-item"
    >
      Asi Bob - 17 June, 2020 year
    </li>
  </ul>
  <h3>
    September
  </h3>
  <ul
    className="birthday-list"
  >
    <li
      className="birthday-item"
    >
      Dark Anna - 10 September, 2018 year
    </li>
  </ul>
</div>
`;
